doc/content/resources/multiapps_chps/chp_5_multiapps.md

```diff
@@ -0,0 +1,215 @@
+
+#### MultiApp Classes in `Level Set App`
+
+The [`LevelSetReinitializationMultiApp`](https://mooseframework.inl.gov/source/multiapps/LevelSetReinitializationMultiApp.html) method is used for front tracking problems, and its mathematical formulations represents the structural solid-void boundary.
+
+### Required Parameters
+
+#### Input Files
+
+The `input_files` parameter is required regardless of MultiApp `type`. The `input_files` parameter specifies the input file for the child application. If the `positions` parameter is used to specify multiple instances of the child application at different locations in the domain, the same input file will be used at each position. It is possible to specify different input files at each position. Meanwhile, if the user has N points specified by either `positions` or `positions_file`, the can provide an array of N input file names for `input_files`. The ith input file name will be used to run the child application at the $i^{th}$ position. In the example below, the input file (name.i) will be used for all the child applications within the MultiApps block named "micro".
+
+!listing
+[MultiApps]
+  [micro]
+    type = TransientMultiApp
+    app_type  = BisonApp
+    positions =   '0.01285  0.0       0
+                   0.01285  0.0608    0
+                   0.01285  0. 1216   0
+                   0.01285  0.1824    0'
+    input_files = name.i
+    execute_on = 'timestep_end'
+  []
+[]
+
+### Optional Parameters
+
+#### Timing of Execution
+
+The `execute_on` parameter specifies when a child application is to be executed, in relation to the parent application timesteps. The most common options are to execute the child application at the beginning of the time step, at the end of the time step, or at the initial condition of the entire simulation. By default, child application execution will occur at the beginning of time-step.  In the previous example, at every time step, the main application will be executed first, then the child applications will be executed at the end of the time step. Some common options of `execute_on` are summarized below:
+
+  1. `TIMESTEP_BEGIN`: Launch child application at the beginning of the time step
+
+  2. `TIMESTEP_END`: Launch child application at the end of the time step
+
+  3. `INITIAL`: Launch child application only at the initial condition to set up the initial conditions for the solve
+
+  4. `NONLINEAR`: This option has limited applications and is used to inject the coupling information into the middle of the Newton solve which will slow down the newton convergence but could speed up the conversion of the entire system.  
```

Comment:
```suggestion
  4. `NONLINEAR`: This option has limited applications and is used to inject the coupling information into the middle of the Newton solve. It will likely slow down the newton convergence, as the contribution to the Jacobian are not considered, but could speed up the convergence of the entire system.  
```